src/schema/artwork/index.js

```diff
@@ -570,20 +518,20 @@ export const artworkFields = () => {
         return partnerLoader(partner.id).catch(() => null)
       },
     },
-    price: {
-      type: GraphQLString,
-    },
+    price: { type: GraphQLString },
+    domesticShipping: amount(
+      ({ domestic_shipping_fee_cents }) => domestic_shipping_fee_cents
```

Comment:
We chatted this on [slack](https://artsy.slack.com/archives/C9ZJYFDNF/p1533108372000187), but I am still not sure if MP is able to get these fees from Gravity, given it's in [`:all`](https://github.com/artsy/gravity/blob/637a58e0172a6ce5dc2d5ac36eedd2aa18ce6578/app/models/domain/artwork.rb#L1013).